spec/project_spec.rb

```diff
@@ -775,6 +851,11 @@ class InvalidErrorHandler; end
       expect(project_instance.get_enabled_features('test_user')).to be_empty
     end
 
+    it 'should call inputs_valid? with the proper arguments in get_enabled_features' do
+      expect(Optimizely::Helpers::Validator).to receive(:inputs_valid?).with({user_id: 'test_user'}, spy_logger, Logger::ERROR)
+      project_instance.get_enabled_features('test_user', 'browser_type' => 'chrome')
```

Comment:
You should also assert that this call returns false. Only asserting that inputs_valid? gets called does not verify that it's result causes the actual method to return false. 
When you assert false, this will sort of confirm. You can also do this in similar tests.